bin/syclcc-clang

```diff
@@ -695,6 +695,8 @@ class syclcc_config:
   def hipsycl_plugin_path(self):
     if sys.platform.startswith('win32'):
       return os.path.join(self.hipsycl_installation_path, "bin", "hipSYCL_clang.dll")
+    elif sys.platform == "darwin":
+      return os.path.join(self.hipsycl_installation_path, "lib", "hipSYCL_clang.dylib")
```

Comment:
The file name is missing the string "lib". Adding this in line 699 fixes the issue on mac, i.e. "**lib**hipSYCL_clang.dylib".

Reply:
Ugh... I feel stupid :( Thanks!

Does that mean that accelerated CPU on mac is really working now?

Reply:
Yes, it's working now!